bucket_map/src/bucket_storage.rs

```diff
@@ -144,7 +147,9 @@ impl BucketStorage {
             let hdr = hdr_slice.as_ptr() as *const Header;
             if hdr.as_ref().unwrap().try_lock(uid) {
                 e = Ok(());
-                self.used.fetch_add(1, Ordering::Relaxed);
+                if !resizing {
+                    self.used.fetch_add(1, Ordering::Relaxed);
+                }
```

Comment:
I think I'm seeing that this `if` block could be outside the `unsafe` block. If not too much of a pain, could you do that change in this PR?